### PR TITLE
Some Details

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/Order.tsx [hunk 1]
```diff
@@ -71,15 +73,9 @@ export const Order = ({ trainInfo, userData }: Props) => {
...
-                  <Typography>
-                    {userData.passengerInformation.name}{' '}
-                    {userData.passengerInformation.surname}
-                  </Typography>
-                  <Typography>
-                    1*Youth(16-26 yrs) ({userData.passengerInformation.discount}
-                    )
-                  </Typography>
-                  <Typography>{userData.passengerInformation.email}</Typography>
+                  <Typography>Name Surname</Typography>
```
Comment (vsvvan) [inline src/components/Order.tsx]: Why did you change this part??

File: src/components/Order.tsx [hunk 2]
```diff
@@ -160,7 +150,7 @@ export const Order = ({ trainInfo, userData }: Props) => {
                 size="large"
                 onClick={() => navigate('/summary')}
               >
-                Pay
+                Payment
               </Button>
             </div>
           </div>
```
Comment (vsvvan) [inline src/components/Order.tsx]: Questions refers to all changes that was made in this file, you revert last changes that were actually correct. Fix this please

File: src/components/App.css [hunk 3]
```diff
@@ -162,6 +162,13 @@ footer li::before {
   margin-left: 7%;
   display: flex;
 }
+.Authors_info {
```
Comment (vsvvan) [inline src/components/App.css:247]: fix name case, take a look at example
.DayStyle {
##
}

File: src/components/App.css [hunk 4]
```diff
@@ -11,6 +11,79 @@
...
 }
+.HeaderMainPage {
+  display: flex;
+  height: 55px;
+  padding: 10px 1% 10px 1%;
+  justify-content: space-between;
+  text-align: center;
+  background: #0773d7;
+}
+.Menu-bar{
```
Comment (vsvvan) [inline src/components/App.css]: should be .MenuBar, the same for others, recheck please

File: src/components/App.css [hunk 5]
```diff
@@ -11,6 +11,79 @@
...
+  display: block;
+  position: absolute;
+  background-color: #0773d7;
+  }
+.Menu-bar ul li:focus-within .Sub-menu-1 ul{
+  display: block;
+  margin: 10px;
+  }
+
+.Sub-menu-1
```
Comment (vsvvan) [inline src/components/App.css]: SubMenu1

File: src/components/HeaderMainPage.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import shoppingCart from '../images/shopping_cart_black_24dp 1.png';
+import React from 'react';
+import Logo from '../images/Logo-header.svg';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '@mui/material';
+
+export const Header = () => {
```
Comment (vsvvan) [inline src/components/HeaderMainPage.tsx]: rename const to the name of the file -> HeaderMainPage and don't forget to rename in place where you use it

File: src/components/Order.tsx (hunk 2 above)
Comment (vsvvan) [reply src/components/Order.tsx]: Rollback all changes that were made in this file

File: src/components/App.css [hunk 7]
```diff
@@ -11,6 +11,79 @@
...
+  padding-right: 1%;}
+.Sub-menu-1 li{
+
+  text-align: left;
+
+}
+.Sub-menu-1 li::before a{
+  display: inline-block;
+  }
+.menu-list{
```
Comment (vsvvan) [inline src/components/App.css]: .MenuList

**Review by vsvvan — CHANGES_REQUESTED** [review]

File: src/components/Header.tsx [hunk 8]
```diff
@@ -8,17 +8,50 @@ export const Header = () => {
...
+
         <Button onClick={() => navigate('/railway/')}>
           <img className="App-logo" src={Logo} alt="Logo" />
         </Button>
+        
+        <nav className='Menu-bar'>
+          <ul>
+          <li><a href="#" className='click'>Ticket Purchase</a>
+            <div className='Sub-menu-1'>
+            <ul className='menu-list'>
```
Comment (vsvvan) [inline src/components/Header.tsx]: reuse menu component from mui library, examples of use are here 
https://mui.com/material-ui/react-menu/#basic-menu

**Review by vsvvan — CHANGES_REQUESTED** [review]

File: src/components/Order.tsx (hunk 1 above)
Comment (ghost) [reply src/components/Order.tsx]: ?? What we need to put there??

Comment (vsvvan) [reply src/components/Order.tsx]: Just remove changes that you did in this file. All changes that you did here are not correct we should use information  from userData object as it was before
You can see what are the difference if you'll go to the order overview page, with your changes there will be Name Surname at user info tab instead of correct name from user.json file (Jakub Liptak or smt like this)

File: src/components/App.css (hunk 3 above)
Comment (ghost) [reply src/components/App.css:247]: ??  same as with Menu-bar?
